src/raven.js

```diff
@@ -324,6 +324,9 @@ Raven.prototype = {
      * @return {Raven}
      */
     captureException: function(ex, options) {
+        // Store the raw exception to pass to errorCallback
+        this._lastCapturedException = ex;
+
```

Comment:
I moved this above the `isError` check because the `errorCallback` would potentially get called with the error object of a previous error for a currently thrown non-error. Another way of doing this would be to bubble the exception itself upwards into `_send`. This seemed easier and until now `this._lastCapturedException` was only used for debugging.